addons/im_livechat/static/src/legacy/public_livechat.xml

```diff
@@ -16,11 +16,11 @@
             </div>
             <div class="o_livechat_rating_reason">
                 <textarea id="reason" placeholder="Explain your note"></textarea>
-                <div class="o_livechat_rating_reason_button">
```

Comment:
Could probably squash this one with "[FIX-IMP] im_livechat: rename Rated user filter into Rated Opperator". (operator takes a single "p" by the way).